cookbooks/podman/resources/service.rb

```diff
@@ -32,7 +32,11 @@
     type "notify"
     notify_access "all"
     environment "PODMAN_SYSTEMD_UNIT" => "%n"
-    exec_start_pre "/bin/rm --force %t/%n.ctr-id"
+    # The exec_start_pre "podman pull" is to workaround the occasional 502 error returned from the registry.
+    exec_start_pre [
+        "/bin/rm --force %t/%n.ctr-id",
+        "-/usr/bin/podman pull #{new_resource.image}"
+      ]
```

Comment:
I'm amazed cookstyle isn't complaining about this indentation.

Reply:
Fixed the formatting.